tools/python/boutiques/bosh.py

```diff
@@ -144,6 +144,9 @@ def parser_executeLaunch():
                         "to cache.")
     parser.add_argument("--provenance", action="store", type=json.loads,
                         help="Append JSON object to the generated record.")
+    parser.add_argument("--no-container", action="store_true",
+                        help="Execution takes place outside of container,"
+                        + " tool is executed in command line.")
```

Comment:
Help text should be "Launch invocation on the host computer, with no container. If 'container-image' appears in the descriptor, it is ignored."